src/main/java/org/spongepowered/api/entity/Entity.java

```diff
@@ -445,4 +445,12 @@ default boolean damage(double damage, DamageSource damageSource) {
      * @param uuid The {@link UUID} to set as notifier.
      */
     void setNotifier(@Nullable UUID uuid);
+
+    /**
+     * Returns whether this entity can see the provided {@link Entity}.
+     *
+     * @param entity The entity to check visibility for
+     * @return {@code true} if this entity can see the provided entity
+     */
+    boolean canSee(Entity entity);
```

Comment:
Why not? I was thinking that this would function similar to methods like `Living#health`, which simply provide convenience method for the underlying API.